src/AzureClient/AzureClient.cs

```diff
@@ -26,6 +26,8 @@ namespace Microsoft.Quantum.IQSharp.AzureClient
     /// <inheritdoc/>
     public class AzureClient : IAzureClient
     {
+        private const string MicrosoftSimulator = "microsoft.simulator";
```

Comment:
As a consequence of this comment: https://github.com/microsoft/iqsharp/pull/601/files#r825100550, this member should no longer be needed.

Reply:
```suggestion
        /// <summary>Returns whether a target ID is meant for quantum execution since not all targets exposed by providers are meant for that. More specifically, the Microsoft provider exposes targets that are not meant for quantum execution and the only ones meant for that start with "microsoft.simulator"</summary>
        private static bool IsQuantumExecutionTarget(string targetId) =>
            AzureExecutionTarget.GetProvider(targetId) == AzureProvider.Microsoft ?
                targetId.StartsWith("microsoft.simulator") :
                true;
```

Reply:
I've added your suggestion but slightly modified. I think using regular Boolean operators is a bit clearer than using the ternary operator. Also the `MicrosoftSimulator` string is still needed elsewhere so that got left in.

Reply:
This modification looks good :)